src/test/java/BaseTest.java

```diff
@@ -1,22 +1,18 @@
-import io.appium.java_client.android.AndroidDriver;
 import org.testng.annotations.AfterMethod;
 import org.testng.annotations.BeforeMethod;
 import steps.StepDefinitions;
 import utils.DriverManager;
 
 public class BaseTest {
-    protected AndroidDriver driver;
-    DriverManager driverManager = DriverManager.getInstance();
-    StepDefinitions stepDefinitions;
+    public StepDefinitions stepDefinitions;
 
     @BeforeMethod
-    public void setUp() throws Exception {
-        driver = (AndroidDriver) driverManager.getDriver();
-        stepDefinitions = new StepDefinitions(driver);
+    public void setUp() {
+        stepDefinitions = new StepDefinitions();
```

Comment:
Do u need to init stepDefs before each method? Do they change somehow from method to method?